### PR TITLE
Update the price for tokens

### PR DESCRIPTION
**Pull request by jinsongo**
Update the price for tokens to be reasonable.

### CODE REVIEW

File: ai/src/main/java/com/instana/dc/ai/impl/llm/LLMDc.java [hunk 1]
```diff
@@ -214,7 +214,7 @@ public void collectData() {
 
             int avgDuration = totalDuration/(requestCount==0?1:requestCount);
             double intervalTokens = (double)totalTokens/LLM_POLL_INTERVAL;
-            double intervalCost = intervalTokens * 0.01;
+            double intervalCost = (intervalTokens/1000) * 0.03;
```
Comment (gyliu513) [inline ai/src/main/java/com/instana/dc/ai/impl/llm/LLMDc.java:217]: Can you update to this logic: $0.01/1k prompt tokens. $0.03/1k sampled tokens.

**Review by liurui-1 — APPROVED** [review]

**Comment by jinsongo** [comment]
@liurui-1 Thank you. I still need a little more changes to calculate cost that depends on the latest change from instrument.

**Comment by liurui-1** [comment]
Please bump up the `version` defined in the `build.gradle` which is the version of maven artifact `com.instana.dc:otel-dc-ai`.

**Comment by jinsongo** [comment]
Don't need it anymore

**Pull request closed without merging**